lib/stdlib/test/epp_SUITE.erl

```diff
@@ -63,16 +63,16 @@ suite() ->
     [{ct_hooks,[ts_install_cth]},
      {timetrap,{minutes,1}}].
 
-all() -> 
+all() ->
     [rec_1, {group, upcase_mac}, include_local, predef_mac,
      {group, variable}, otp_4870, otp_4871, otp_5362, pmod,
      not_circular, skip_header, otp_6277, otp_7702, otp_8130,
      overload_mac, otp_8388, otp_8470, otp_8562,
      otp_8665, otp_8911, otp_10302, otp_10820, otp_11728,
      encoding, extends, function_macro, test_error, test_warning,
-     otp_14285, test_if, source_name, otp_16978, otp_16824, scan_file].
+     otp_14285, test_if, source_name, otp_16978, otp_16824, scan_file, file_macro].
```

Comment:
Have you tried to run this test suite? `file_macro/1` needs to be exported or the test case will fail.

Reply:
Exported file_macro, and have run the test now and it passes, sorry about that.